packages/editor/src/lib/editor/Editor.ts

```diff
@@ -709,21 +708,33 @@ export class Editor extends EventEmitter<TLEventMap> {
 			this._tickManager.start()
 		})
 
-		const checkLicenseKey = async (licenseKey: string | undefined) => {
-			if (!featureFlags.enableLicensing.get()) return
+		this.licenseManager = new LicenseManager(licenseKey)
 
-			const licenseManager = new LicenseManager()
-			const watermarkManager = new WatermarkManager(this)
-			const license = await licenseManager.getLicenseFromKey(licenseKey)
-			watermarkManager.checkWatermark(license)
+		const youWouldntStealAWhiteboard = () => {
```

Comment:
just making sure: no more feature flag? official launch?

Reply:
My mistake (removed this bit now anyway)